### DISCUSSION TITLE
Anchors not getting rendered

### DISCUSSION THREAD

**Discussion by WendellAdriel** [spatie/laravel-markdown#65] [Q&A] [answered]
I have this in my configuration file

```php
/*
 * When enabled, anchor links will be added to all titles
 */
'add_anchors_to_headings' => true,
```

But as you can see here: https://wendelladriel.com/blog/laravel-aaas-actions-as-a-service
The titles don't have anchors at all, am I missing something that needs to be done so the anchors can be created?

**Comment by sebastiandedeyne** [comment, marked as answer]
I believe the "anchors" are the `id` attribute on the headers. Looking at the source code of AnchorHeadingRenderer that's all this package does.

https://github.com/spatie/laravel-markdown/blob/4f05cac35606ac9c8a3be77a29636f368b3805b8/src/Renderers/AnchorHeadingRenderer.php

Would a PR for an additional config option that also renders `a` tags though. (Additional config option to avoid breaking changes)

**Reply by WendellAdriel** [reply]
Oh, got it, thanks @sebastiandedeyne!!!
I knew I was missing something!
I'll take a look to see if I can create a PR to add a new config for that, thanks!

**Reply by WendellAdriel** [reply]
@sebastiandedeyne I just created a PR for that: https://github.com/spatie/laravel-markdown/pull/66
